pkg/storage/secret/encryption/encrypted_value_store_test.go

```diff
@@ -198,7 +198,11 @@ func TestEncryptedValueStoreImpl(t *testing.T) {
 	})
 }
 
-func TestStateMachine(t *testing.T) {
+func TestIntegration_StateMachine(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
```

Comment:
I would suggest using `tests.SkipIntegrationTestInShortMode(t)`, which not only does what you're doing here, but also checks that the calling test is indeed an integration test.